From 6fd492e5d0db2f6c106452a3efdb22b557f23a60 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michael=20La=C3=9F?= <bevan@bi-co.net>
Date: Wed, 21 Jun 2023 00:54:38 +0200
Subject: [PATCH] Update to Gradle 8.1.1

Gradle 8.1 allows building this project with Java 20.

Spring Boot needs to be updated as well to support Gradle 8.1. This
change updates Spring Boot to the latest available version 3.1.0.
---
 gradle.properties       | 2 +-
 lib/ac-api/build.gradle | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/gradle.properties b/gradle.properties
index 1010a1a..901b57d 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -15,5 +15,5 @@ commonsLang3Version=3.7
 commonsCsvVersion=1.8
 gsonVersion=2.8.6
 picocliVersion=4.6.2
-springBoot=2.6.1
+springBoot=3.1.0
 acdasmVersion=0.4.0
diff --git a/lib/ac-api/build.gradle b/lib/ac-api/build.gradle
index a2e5c4d..2ee7301 100644
--- a/lib/ac-api/build.gradle
+++ b/lib/ac-api/build.gradle
@@ -41,12 +41,12 @@ javadoc {
 }
 
 task javadocJar(type: Jar) {
-    classifier = 'javadoc'
+    archiveClassifier = 'javadoc'
     from javadoc
 }
 
 task sourcesJar(type: Jar) {
-    classifier = 'sources'
+    archiveClassifier = 'sources'
     from sourceSets.main.allSource
 }
 
-- 
2.41.0

